nospoiler/nospoiler.py

```diff
@@ -323,40 +376,47 @@ async def warn(self, ctx: commands.Context) -> None:
-        )
-        await ctx.send("Spoiler warning message has been reset.")
+    async def warnmessage(
+        self, ctx: commands.Context, *, message: TagscriptConverter
+    ) -> None:
+        """
+        Set the spoiler warning message.
 
-    @nospoiler.command()
-    async def embed(self, ctx: commands.Context) -> None:
-        """Toggle the spoiler warning message embed."""
-        if not ctx.bot_permissions.embed_links:
-            self.log.info(
-                f"I cannot toggle the spoiler warning message embed because I don't have embed_links permission in {ctx.channel.mention}."
-            )
-            return await ctx.send(
-                "I don't have embed_links permission to let you toggle the spoiler warning message embed."
+        (Supports Tagscript)
+
+        **Blocks:**
+        - [Assugnment Block](https://phen-cogs.readthedocs.io/en/latest/tags/tse_blocks.html#assignment-block)
+        - [If Block](https://phen-cogs.readthedocs.io/en/latest/tags/tse_blocks.html#if-block)
+        - [Embed Block](https://phen-cogs.readthedocs.io/en/latest/tags/parsing_blocks.html#embed-block)
+        - [Command Block](https://phen-cogs.readthedocs.io/en/latest/tags/parsing_blocks.html#command-block)
+
+        **Variable:**
+        - `{server}`: [Your guild/server.](https://phen-cogs.readthedocs.io/en/latest/tags/default_variables.html#server-block)
+        - `{member}`: [Author of the message.](https://phen-cogs.readthedocs.io/en/latest/tags/default_variables.html#author-block)
+        - `{color}`: [botname]'s default color.
+
+        **Example:**
+        ```
+        {embed(description):{member(mention)} You cannot send spoilers here.}
+        {embed(color):{color}}
+        ```
+        """
+        if message:
+            await self.config.guild(ctx.guild).spoiler_warn_message.set(message)
+            await ctx.send(
+                f"Sucessfully changed the warn message: \n {box(f'{message}', lang='json')}"
             )
-        spoiler_warn_message_embed = await self.config.guild(
-            ctx.guild
-        ).spoiler_warn_message_embed()
-        if spoiler_warn_message_embed:
-            await self.config.guild(ctx.guild).spoiler_warn_message_embed.set(False)
-            await ctx.send("Spoiler warning message embed is now disabled.")
         else:
-            await self.config.guild(ctx.guild).spoiler_warn_message_embed.set(True)
-            await ctx.send("Spoiler warning message embed is now enabled.")
+            await self.config.guild(ctx.guild).spoiler_warn_message.clear()
+            await ctx.send("Successfully resetted the warn message.")
+
+    @nospoiler.command()
+    async def delete(self, ctx: commands.Context, amount: commands.Range[int, 0, 100]):
+        """
+        Change when the warn message get's deleted.
+        """
+        await self.config.guild(ctx.guild).delete_after.set(amount)
+        await ctx.send(f"The delete after is now {amount}.")
```

Comment:
Due to the limit from discord and avoiding rate limit, should be from  5 sec and 120 sec.